src/components/graph/menus/network-modification-node-editor.js

```diff
@@ -446,7 +473,47 @@ const NetworkModificationNodeEditor = () => {
             studyUuid,
             currentTreeNode?.id,
             [...selectedItems.values()].map((item) => item.uuid)
-        ).then();
+        )
+            .then()
+            .catch((errmsg) => {
+                console.error('doDeleteModification error:', errmsg);
+                snackError('', 'errDeleteModificationMsg');
+            });
+    };
+
+    const doCopyModification = () => {
+        // just memorize the list of selected modifications
+        let newCopySet = new Set();
+        selectedItems.forEach((item) => {
+            if (item.type !== 'GROOVY_SCRIPT') {
```

Comment:
since "copy" button is disabled when a groovy script is selected, do we need this ?

Reply:
this code does not exist anymore (US has changed)